package.json

```diff
@@ -90,7 +90,7 @@
     "flexboxgrid": "^6.3.0",
     "immutable": "^3.8.1",
     "lodash": "^4.5.1",
-    "lzma-native": "^1.5.2",
+    "lzma-native": "^2.0.2",
```

Comment:
...so we should stop shrinkwrapping the `node-gyp` version now? (which I guess also means scripts/ci/ensure-npm-shrinkwrap-versions.sh becomes defunct)